### ISSUE TITLE
Elements that act as buttons but use non-button tags (e.g., span, div, a) should include a role="button" attribute.

### ISSUE THREAD

**Issue by bsylwek** [joomla/joomla-cms#45596]
### Steps to reproduce the issue
1. How can I solve this problem? I have the standard Cassiopeia template.
 Please go to scan https://accessibe.com/accessscan?website=bip.dps2.vernet.pl

### Expected result

 10 Code snapshots of failed elements

    <i class="_access-icon _access" style="width: 50px; height: 50px; font-size: 40px; line-height: 45px; text-indent: 5px; background-color: transparent; color: rgb(255, 255, 255); bottom: 50px; left: 0px; z-index: 9999; opacity: 1;" title="Ustawienia ...

    <i class="_menu-close-btn _menu-btn " title="Zamknij">X</i>

    <i class="_menu-reset-btn _menu-btn " title="Wyczyść">♲</i>

    <li data-access-action="increaseText">Zwiększ rozmiar tekstu</li>

    <li data-access-action="decreaseText">Zmniejsz rozmiar tekstu</li>

    <li data-access-action="increaseTextSpacing">Zwiększ odstępy w tekście</li>

    <li data-access-action="decreaseTextSpacing">Zmniejsz odstępy w tekście</li>

    <li data-access-action="invertColors" title="Hotkey: ctrl+alt+i">Odwróć kolory</li>

    <li data-access-action="grayHues" title="Hotkey: ctrl+alt+g">Szare odcienie</li>

    <li data-access-action="underlineLinks" title="Hotkey: ctrl+alt+u">Podkreślaj łącza</li>

### Device and system info

Joomla 4.4.13

* What device were you using? (eg, phone, laptop)

* What web browser were you using? (eg, Chrome, Safari)

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:
